### PR TITLE
Fix unix OS detection.

### PR DESCRIPTION
**Pull request by mfilippov**
Mono returns PlatformID.Unix on Linux and macOS. It's not enough for detect OS version. I use native call uname from libc for fix it.

### CODE REVIEW

File: src/BenchmarkDotNet.Core/Portability/RuntimeInformation.cs [hunk 1]
```diff
@@ -263,5 +265,31 @@ public JitModule(string name, string version)
                 Version = version;
             }
         }
+
+        [DllImport("libc", SetLastError=true)]
+        private static extern int uname(IntPtr buf);
+
+        private static string GetSysnameFromUname()
+        {
+          var buf = IntPtr.Zero;
```
Comment (AndreyAkinshin) [inline src/BenchmarkDotNet.Core/Portability/RuntimeInformation.cs]: @mfilippov, could you increase the indent here?

Comment (mfilippov) [inline src/BenchmarkDotNet.Core/Portability/RuntimeInformation.cs]: fixed

**Comment by mfilippov** [comment]
Build failed, looks like NuGet server problem not depended with my change.

**Comment by AndreyAkinshin** [comment]
@mfilippov, thanks!

**Pull request merged**